src/ServiceInsight.Tests/packages.config

```diff
@@ -5,6 +5,7 @@
   <package id="Caliburn.Micro.Core" version="2.0.2" targetFramework="net45" />
   <package id="NSubstitute" version="1.10.0.0" targetFramework="net45" />
   <package id="NUnit" version="3.2.1" targetFramework="net45" />
+  <package id="ObservablePropertyChanged" version="0.1.3" targetFramework="net452" developmentDependency="true" />
```

Comment:
If not having the dependency listed here is problematic then I'm not strongly opposed to it.

@HEskandari before doing that, would you mind investigating if you can solve your local tooling (Resharper) problem in another way? E.g. by switching off that warning by default?